pkg/console/controllers/managedclusters/controller.go

```diff
@@ -226,6 +226,21 @@ func (c *ManagedClusterController) SyncManagedClusterList(ctx context.Context) (
 			continue
 		}
 
+		// Get Claim values in a map
+		claims := util.ClaimsMap(&managedCluster)
+
+		// Omit clusters that have the following product name
+		omittedProductCluster := []string{"AKS", "EKS", "IKS", "ROKS"}
+		if s, ok := claims["product.open-cluster-management.io"]; ok && util.SliceContains(omittedProductCluster, s) {
+			klog.V(4).Infof("Skipping managed cluster %v, based on vendor label, cluster unsupported", clusterName)
+			continue
+		}
+
+		// Omit any clusters with version less than ocp 4.0
+		if s, ok := claims["version.openshift.io"]; ok && s < "4.0" {
```

Comment:
I did test this logic and because the string "3" is < the string "4" is does work.  However I will look at your alternative since it does not really make since to use math operators on a string.

Reply:
So researching this, both methods are valid and both use the lexical order so they are essentially the same AFAIK.  I will leave the code as is for readability, if that works for you.